Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
```

Comment:
This works just fine here, but it is a good practice to avoid mutating (changing) values when it's not needed. 
For example: If it was a much longer function (let's say hundreds of lines) you would need to check all the places where isPrimeInd is re-assigned new value
and remember about all of them to completely understand what happens in the function. 
Here you can very easily completely avoid isPrimeInd. E.g. {isprimeInd = false; break;} can be simply replaced with {return false;}. Try to get rid of this variable